test/orange/OrangeGeant.test.cc

```diff
@@ -198,6 +198,39 @@ TEST_F(SimpleCmsGeantTest, trace)
     }
 }
 
+//---------------------------------------------------------------------------//
+class TilecalPlugTest : public GeantOrangeTest
+{
+    std::string geometry_basename() const final { return "tilecal-plug"; }
+};
+
+TEST_F(TilecalPlugTest, trace)
+{
+    if (0)
```

Comment:
Maybe add a TODO here to explain why this block is `if`-ed out, on the off chance that it is forgotten about in the future.

Reply:
Actually the test passes now. Good catch!